### ISSUE TITLE
Can I query isFuture / isOngoing / isPast via graphql?

### ISSUE THREAD

**Issue by FreekVR** [studioespresso/craft-date-range#25]
I'm trying to query only past/future/ongoing elements via the graphql-api, but I can't figure out how to do it through query arguments

Eg this doesn't work
```
query events {
  entries(
    section: "educationAndEvents", 
    isFuture: ["dateRange"]
  ) {
    id
    title
    dateRange {
      start
      isFuture
    }
  }
}
```

I've also tried messing with the "dateRange"-argument (the name of our field) but wasn't able to get that to work either.

Is there any way to do this or would this be a feature request :)?

**Comment by janhenckens** [comment]
Hey @FreekVR, that would be cool, let look into it!

**Comment by MrHesus** [comment]
Hey @janhenckens I made a pull request for this feature:

https://github.com/studioespresso/craft-date-range/pull/28

**Comment by janhenckens** [comment]
Awesome @MrHesus! Looks good, I'll merge & have a look.

**Comment by janhenckens** [comment]
Just released 2.4.0, which includes this!
